test/journeys/lib/test-helpers/space-widget/messaging.js

```diff
@@ -134,23 +137,19 @@ export function flagMessage(testObject, messageToFlag) {
 export function removeFlagMessage(testObject, messageToUnflag) {
   testObject.browser.waitForExist(elements.pendingActivity, 15000, true);
   testObject.browser.waitUntil(() =>
-    testObject.browser.element(elements.lastActivityText).getText() === messageToUnflag, 'message was not found');
+    testObject.browser.element(elements.lastActivityText).getText() === messageToUnflag, 1500, 'message was not found');
 
   testObject.browser.waitUntil(() => testObject.browser
-    .element(elements.lastActivity)
-    .element(elements.highlighted)
-    .element(elements.flagButton)
-    .isVisible(), 'message was not flagged');
+    .element(`${elements.lastActivity} ${elements.highlighted} ${elements.flagButton}`)
+    .isVisible(), 1500, 'message was not flagged');
```

Comment:
Instead of selecting using `.element()` and then checking `.isVisible()`, which is 2 api calls. Can we pass the selector directly into `.isVisible()`?

Reply:
I was hoping to make a big PR to switch all of our journey tests to this method if you prove that it will stabilize more. For this one, I wanted to match what was already in place, just remove the multiple elements.